CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -0,0 +1,209 @@
+# Release notes for 1.3.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.2.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Adds mappings for PV access modes to new CSI access modes: `SINGLE_NODE_SINGLE_WRITER` and `SINGLE_NODE_MULTI_WRITER`. ([#151](https://github.com/kubernetes-csi/external-resizer/pull/151), [@chrishenzie](https://github.com/chrishenzie))
```

Comment:
This one should be a feature, not cleanup/flake.

Reply:
but it has no impact on any of resizer features. It is entirely because the whole mechanism of getting capabilities was changed. This PR adds nothing new to resizer as such.

Reply:
It allows resizer to support drivers that support the new modes/capabilities right?

Reply:
It also enables the resizer to process the new RWOP access mode on PVs right?

Reply:
yes sure. but I am not sure if it is a new feature as such in resizer. but anyways - I moved this line under features section.